apps/web/scripts/kernel/utils/handler.ts

```diff
@@ -32,3 +28,8 @@ export const getNetwork = (requestType: RequestType) => {
 	if (typeName.includes('ON_SOLANA')) return Networks.solana;
 	else if (typeName.includes('ON_SUI')) return Networks.sui;
 };
+
+export const triggerActionToGetPrivateKey = async () => {
+	console.log('bug');
+	return Uint8Array.from([1, 2, 3]);
+};
```

Comment:
check this method

Reply:
> check this method

this is a bug that need fixing, i think we should put it there to remind fixing it